src/main/kotlin/org/jetbrains/plugins/bsp/server/tasks/CollectProjectDetailsTask.kt

```diff
@@ -271,44 +266,26 @@ public class CollectProjectDetailsTask(project: Project, private val taskId: Any
   }
 
   private suspend fun addBspFetchedPythonSdks() {
```

Comment:
prob we should check the existance of the extension during calcualtion of the sdks - if it doesnt exeist there is no point in calculating pyton sdks